sources/packages/web/src/components/aest/AssessApplicationOfferingChangeRequestModal.vue

```diff
@@ -73,6 +83,7 @@ export default defineComponent({
       resolvePromise(false);
     };
     const assessChange = async () => {
+      loading.value = true;
```

Comment:
Are we implementing loading or not? I thought that we were not going ahead with this right now.
If we are, if we are setting `loading.value = true` here and it fails the validation in line 89 should we not set it to false?
I believe that the idea would be to control the loading either internally in the modal or externally. Are we controlling on both ends?

Reply:
I checked it using `setTimeout` to delay the dialog close event and I can see the loader till the promise is resolved.  
Plus, what I feel is that since the model component gets destroyed after close happens, we do not actually need to set the loading to false again explicitly, right?

Also, IMO, we can control the loading both internally and externally.

Reply:
Updated it as per the discussion in call.